curve25519-dalek/src/scalar.rs

```diff
@@ -1120,7 +1141,21 @@ impl Scalar {
         UnpackedScalar::from_bytes(&self.bytes)
     }
 
-    /// Reduce this `Scalar` modulo \\(\ell\\).
+    /// Transform this `Scalar` to a `WideScalar` for faster arithmetic with lazy modulus reduction
+    pub fn to_wide(&self) -> WideScalar {    
+        self.unpack().to_wide()
+    }
+
+    /// Build a `Scalar` from a `WideScalar`
+    pub fn from_wide(wide: WideScalar) -> Self {
+        wide.to_scalar().pack()
+    }
+
+    /// Wide `Scalar` multiply accumulate
+    pub fn mul_acc(acc: &mut WideScalar, lhs: &Self, rhs: &Self) {
+        acc.mul_acc(&lhs.unpack(), &rhs.unpack());
+    }
+ 
```

Comment:
You mistakenly removed the previous comment related to the reduce function below